settings/src/components/account-status.js

```diff
@@ -36,6 +41,19 @@ export default function AccountStatus() {
 				${ backupCodesEnabled ? '' : 'not' }
 				verified your backup codes for two-factor authentication.`;
 
+	let svnBodyText = '';
+	if ( svnPasswordRequired || svnPasswordSet ) {
+		if ( svnPasswordRequired ) {
+			svnBodyText = svnPasswordSet
+				? 'You have an active SVN password set. You can request a new one at any time.'
```

Comment:
Only reviewing wording here:
* Do we need to explicitly define like "SVN (Subversion)"?
* Could be briefer and more direct, like "A SVN password is set" (or active)
* Makes me wonder if there's such a thing as an inactive SVN password?

Reply:
I kind of hope that those who need SVN access would know what SVN is, but, it's easy so why not?

> Makes me wonder if there's such a thing as an inactive SVN password?

There isn't, currently. An 'inactive SVN password' would be an account that doesn't have a svn password set, or that we expired it (some time in the future).

Reply:
Noting the text was simplified significantly.

> Do we need to explicitly define like "SVN (Subversion)"?

I wonder if people know it more as `SVN` or `Subversion`.. I actually suspect it's more the former in the WordPress sphere.

looking at other WordPress documentation:
 - https://codex.wordpress.org/Using_Subversion doesn't call it SVN, but refers to our SVN repo's.
 - https://make.wordpress.org/core/handbook/contribute/svn/ refers to 'Subversion' twice, and SVN a bunch more.

I've experimented with `<abbr>` and `Subversion (SVN)`, and have settled on using SVN everywhere but calling it `Subversion (SVN)` in the first sentence on the page. Additionally, linking to the plugins handbook, see [98fcc52](https://github.com/WordPress/wporg-two-factor/pull/280/commits/98fcc52adf7e0e6c9e88d95168d3f89109268869)